index.src.html

```diff
@@ -2530,6 +2548,11 @@
   [EnforceRange] unsigned long long duration;  // microseconds
   [EnforceRange] long long timestamp;          // microseconds
 };
+
+enum AlphaOption {
+  "keep",
+  "discard",
+};
 </xmp>
 
 ### Internal Slots ###{#videoframe-internal-slots}
```

Comment:
Can you make an internal slot that is [[has alpha]] and make the hasAlpha attr simply proxy that. For example:
https://w3c.github.io/webcodecs/#dom-audiodecoder-state

Reply:
This is done.